src/pynwb/image.py

```diff
@@ -212,23 +212,49 @@ class IndexSeries(TimeSeries):
     array indicates when that image was displayed.
     '''
 
-    __nwbfields__ = ('indexed_timeseries',)
+    __nwbfields__ = (
+        "indexed_timeseries",
+        "indexed_images"
+    )
```

Comment:
Was this missing `nwbfield` a bug? Should tests be added for it?

Reply:
I'm not sure. Actually let me try to remove this and see if everything still works